RT-OneEntry/ListTriggerReadThrough.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Redis;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client;
+using Newtonsoft.Json.Linq;
+using StackExchange.Redis;
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+    public static class ListTriggerReadThrough
+    {
+        public const string localhostSetting = "redisLocalhost";
+        private static readonly IDatabase cache = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(localhostSetting)).GetDatabase();
+
+        [FunctionName(nameof(ListTriggerReadThroughFunc))]
+        public static async Task ListTriggerReadThroughFunc(
+            [RedisPubSubTrigger(localhostSetting, "__keyevent@0__:keymiss")] string listEntry, [CosmosDB(
+                            databaseName: "databaseName",
+                            containerName: "containerName",
+                            Connection = "Endpoint" )]CosmosClient input,
+            ILogger logger)
+        {
+            Container db = input.GetDatabase("databaseName").GetContainer("containerName");
```

Comment:
Line 41 - 47 are the same for your WiriteBack function, should we create a method that takes care of this so we have less code duplication? That method could also help with writing unittests.